<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-97PDBKT" name="tutorial1.test2" tests="4" failures="0" timestamp="08 Mar 2020 14:17:25 GMT" time="0.048" errors="1">
  <testcase name="sixthtest" time="0.000" classname="tutorial1.test2">
    <error type="org.testng.TestNGException" message="
Data Provider public static java.lang.String tutorial1.test2.data() must return either Object[][] or Object[] or Iterator&lt;Object[]&gt; or Iterator&lt;Object&gt;, not class java.lang.String">
      <![CDATA[org.testng.TestNGException: 
Data Provider public static java.lang.String tutorial1.test2.data() must return either Object[][] or Object[] or Iterator<Object[]> or Iterator<Object>, not class java.lang.String
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:156)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- sixthtest -->
  <testcase name="fourthtest" time="0.008" classname="tutorial1.test2"/>
  <testcase name="fifthtest" time="0.020" classname="tutorial1.test2"/>
  <testcase name="thirdtest" time="0.020" classname="tutorial1.test2"/>
</testsuite> <!-- tutorial1.test2 -->
